nextcloud/android
# Bugfix/Make cancelation work again for Android >= 11 via Worker

File cancellation was broken for the "new" FilesUploadWorker that's used for uploads in Android >= 11. 
Uses the same cancel function as before and should work like in Android <= 10. 

To test: 
Upload (big) file, go to uploads and press on the "X" for an upload. Now it should work as expected, while before it just did nothing. 

Do I need tests here?
